packages/twenty-front/src/modules/object-record/utils/generateEmptyFieldValue.ts

```diff
@@ -49,8 +49,21 @@ export const generateEmptyFieldValue = (
       };
     }
 
-    case FieldMetadataType.MultiSelect:
-    case FieldMetadataType.Select: {
+   // Handling MultiSelect and Select in the default case
+    default: {
+      if (
+        fieldMetadataItem.type === FieldMetadataType.MultiSelect ||
+        fieldMetadataItem.type === FieldMetadataType.Select
+      ) {
+        const defaultSelectValue = {
+          color: 'DEFAULT_COLOR', // Replace with the actual default color
```

Comment:
let's take the first possible option color and first select option if the field is non nullable and leave it null if nullable